config/initializers/doorkeeper.rb

```diff
@@ -18,6 +18,8 @@ def self.optional
 
   realm "Panoptes"
 
+  grant_flows ["authorization_code", "client_credentials", "implicit", "password"]
```

Comment:
we're using implicit for non zooniverse.org domains and a custom implementation of devise session -> token for PFE.